tails_files/install.sh

```diff
@@ -61,14 +83,19 @@ if $ADMIN; then
   echo "# HidServAuth lines for SecureDrop's authenticated hidden services" | cat - $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths > $ADDITIONS
   if [[ -d "$HOMEDIR/.ssh" && ! -f "$HOMEDIR/.ssh/config" ]]; then
     # create SSH host aliases and install them
-    SSHUSER=$(zenity --entry --title="Admin SSH user" --window-icon=$INSTALL_DIR/securedrop_icon.png --text="Enter your username on the App and Monitor server:")
+    # parse prod-specific.yml YAML into Bash variables
+    eval $(parse_yaml $ANSIBLE/prod-specific.yml)
+    if [[ -z $ssh_users ]]; then
+        echo "Error: ssh_users is not defined in prod-specific.yml." 1>&2
+        exit 1
+    fi
     cat > $INSTALL_DIR/ssh_config <<EOL
 Host app
   Hostname $APPSSH
-  User $SSHUSER
+  User $ssh_users
 Host mon
   Hostname $MONSSH
-  User $SSHUSER
+  User $ssh_users
```

Comment:
I ask because if `ssh_users` has more than one entry, the SSH config auto-setup will fail because SSH expects a single username here. This is sufficiently confusing that, if we cannot change `ssh_users` to `ssh_user` due to the "merge conflict on the Admin Workstation during update" issue, we should probably document this clearly.